lib/passkit/certificate_sources/factory.rb

```diff
@@ -0,0 +1,9 @@
+class Passkit::CertificateSources::Factory
+  def self.find_source(site_id)
+    if Passkit.configuration.use_database_for_certificates
+      Passkit::CertificateSources::Database.new(site_id)
```

Comment:
We can built this source in Talkable and pass source (instead of site_id) as an argument.